src/Host/Host.php

```diff
@@ -209,31 +210,49 @@ public function multiplexing(bool $multiplexing = true)
         return $this;
     }
 
+    /**
+     * @return bool
+     */
+    public function isBecomeSetHome()
+    {
+        return $this->becomeSetHome;
+    }
+
+    /**
+     * @param bool $becomeSetHome
```

Comment:
would be great to have more docs on this "setter", so people know why/when this switch will be usefull.